Code/Framework/AzCore/AzCore/Math/MathStringConversions.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/functional.h>
+#include <AzCore/std/string/conversions.h>
+#include <AzCore/std/string/string.h>
+
+namespace AZ
+{
+    class Matrix4x4;
+    class Quaternion;
+    class Transform;
+    class Vector2;
+    class Vector3;
+    class Vector4;
+}
+
+namespace AZStd
+{
+    //! Prints a Vector2 with precision to 8 decimal points
+    void to_string(string& str, const AZ::Vector2& value);
```

Comment:
I believe these names should be camel case as per core coding standards: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md

Reply:
Which names? The variables are technically camel case by virtue of being single words and to_string while not, is the existing standard for the API.

Reply:
AZStd follows the std naming conventions in all places for public methods. Hence Rajiv has picked the correct method names in this case.